src/VisualStudio/Core/Def/Implementation/TaskList/ProjectExternalErrorReporter.cs

```diff
@@ -135,26 +135,33 @@ private DiagnosticData CreateDocumentDiagnosticItem(ExternalError error)
 
             var line = error.iLine;
             var column = error.iCol;
-            /* TODO: make work again
```

Comment:
lol :)  i'm glad we could get back around to this.

Reply:
Yeah this is my fault. In 7e10f449b4d6fa732242c194b962c78622e199ea I was gutting our IHostDocument stuff, and trying to fix all of the code at once was going to be untenable since I needed to test if the core stuff was working. So I commented out some stuff and was going to fix it up before merging. Somehow we missed this one. (It wasn't the only one.)